lua/apisix.lua

```diff
@@ -41,6 +45,7 @@ function _M.init_worker()
     require("apisix.service").init_worker()
     require("apisix.consumer").init_worker()
     require("apisix.heartbeat").init_worker()
+    require("apisix.admin.init").init_worker()
```

Comment:
The admin API will always run whether I disabled the admin API in yaml?

Reply:
fixed already

Reply:
@moonming I suggest `enable_admin` to be `enable_admin_api` for clarity.
https://github.com/apache/apisix/blob/f31bef426ef89af950e43058dad2143baec41612/conf/config-default.yaml#L31